specs/centos_ci.adoc

```diff
@@ -0,0 +1,135 @@
+// vim: tw=79
+
+= CentOS CI
+
+We need to make sure that new code doesn't break existing codebase and check
+that new features works as early as possible. This can be done by running
+acceptance integration tests automatically, using CentOS CI infrastructure.
+
+== Problem description
+
+CentOS CI
+will be configured to launch acceptance integration test run for every new
+nightly rpm build from tendrl nightly copr project. As set of implemented
```

Comment:
We should.

However, I'm wondering how this will actually play out. We need to be able to nightly test all the specifications that are in progress. This means that we need to set some way by which the packages can be built for specific branches nightly. These branches would be spread across multiple component, linked to from the specification issue. My thought is that we need to specify the branch name to be used in every repository across the project that contributes towards a specification. The name can be something as straightforward as `spec-$(issue_id)`.

In any case, would such builds be possible with copr? If not, we should be able to use the CentOS CI infrastructure itself to build artifacts and repositories and use them for individual specifications' nightly test runs.

Need inputs from @nthomas-redhat here.